sqlmesh/core/console.py

```diff
@@ -252,20 +260,24 @@ def show_model_difference_summary(
         if context_diff.added:
             added_tree = Tree(f"[bold][added]Added Models:")
             for model in context_diff.added:
-                added_tree.add(f"[added]{model}")
+                if model not in ignored_snapshot_names:
```

Comment:
[Nit] So if all added models happened to be ignored we'd still show the `Added Models` header.

Reply:
Thanks for catching these